testing/windows_local_audio_client.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import struct
+import numpy as np
+import time
+import requests
+import pyaudiowpatch as pyaudio
+
+def translator():
+    DOCKER_IP = "http://0.0.0.0:8000/"
```

Comment:
This should be a global or a parameter passed into the translator function because the IP address of the server is not static and may need to be changed by the user.
Consider allowing the user to pass in the addr/port as command line arguments.